cm2350/mpc5674.py

```diff
@@ -740,9 +808,11 @@ def externalResetHandler(self):
     def init(self):
         logger.info('INIT')
 
+        ram_config = self.get_project_config('project.MPC5674.SRAM')
```

Comment:
is this redundant?  
```
        ram_config = self.get_project_config('project.MPC5674.SRAM')
```

should `project` be limited here?

Reply:
limited in what way?